config/secrets.defaults.yml

```diff
@@ -0,0 +1,7 @@
+---
+test:
+  scvmm_defaults: &scvmm_defaults
+    hostname: scvmm1111.manageiq.com
```

Comment:
should this be `scvmm.example.com` to be consistent with the pattern in the other repos?

Reply:
I just copied what was in the spec file already but sure